SuperBuild/External_python-ShapeVariationAnalyzer-requirements.cmake

```diff
@@ -42,15 +42,27 @@ if(NOT Slicer_USE_SYSTEM_${proj})
 
   set(requirements_file ${CMAKE_BINARY_DIR}/${proj}-requirements.txt)
   file(WRITE ${requirements_file} [===[
+  # [packaging]
   joblib==0.16.0 --hash=sha256:d348c5d4ae31496b2aa060d6d9b787864dd204f9480baaa52d18850cb43e9f49
```

Comment:
The running of the utility in this manners has specifically only updated what it needed rather than to all latest versions. Were you expecting this/this what you intended? For example joblib==0.16.0 is valid for scikit-learn==1.0.2 as the [minimum version allowed is joblib 0.11](https://github.com/scikit-learn/scikit-learn/blob/1.0.2/sklearn/_min_dependencies.py#L20), even though technically there is a newer version available, [joblib==1.1.0](https://pypi.org/project/joblib/1.1.0/). In scikit-learn `main` the [minimum version allowed for joblib is now 1.0.0](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/_min_dependencies.py#L17). If you were to update `joblib` to latest version now, you might not have to update it upon the next scikit-learn version release assuming it maintains the joblib 1.0.0 minimum.

Reply:
The hint here should have been ` # [joblib]` and not ` # [packaging]`